Content.Server/Spreader/KudzuSystem.cs

```diff
@@ -107,6 +107,7 @@ private void SetupKudzu(EntityUid uid, KudzuComponent component, ComponentStartu
     /// <inheritdoc/>
     public override void Update(float frameTime)
     {
+        var appearanceQuery = GetEntityQuery<AppearanceComponent>();
```

Comment:
@metalgearsloth i wonder why you did this, is it more performant than TryComp<AppearanceComponent> inside the loop?